components/ws-manager-api/go/config/config.go

```diff
@@ -118,6 +118,7 @@ type Configuration struct {
 // AllContainerConfiguration contains the configuration for all container in a workspace pod
 type AllContainerConfiguration struct {
 	Workspace ContainerConfiguration `json:"workspace"`
+	PVC       PVCConfiguration       `json:"pvc"`
```

Comment:
Not sure this is the right place. I'd move this to `Configuration` for now.
With the introduction of [workspace classes](https://github.com/gitpod-io/gitpod/pull/9238) the `AllContainerConfiguration` struct goes away.